ports/powerpc/powerpc.lds

```diff
@@ -0,0 +1,10 @@
+SECTIONS
+{
+        . = 0;
+        _start = .;
+        .head : {
+                KEEP(*(.head))
+        }
+
+        . = 0x1700;
```

Comment:
maybe add a comment as to why this number / what this line does?

Reply:
yep, it for putting it right after the exception vectors. I'll add the comment in the code